training/training.pages.inc

```diff
@@ -5,16 +5,601 @@
  */
 
 /**
- * Renders example form.
+ * Renders a page Example CTools Javascript.
  *
+ * @return string
+ */
+function training_js_callback() {
+  global $user;
+  drupal_add_js(drupal_get_path('module', 'training') . '/js/training.js');
+  $settings = array(
+    'user_id' => $user->uid,
+    'random_num' => rand(),
+  );
+  drupal_add_js($settings, 'setting');
+
+  return t('User ID = %uid', array(
+    '%uid' => $user->uid,
+  ));
+}
+
+/**
+ * Renders a page Example CTools Ajax.
+ *
+ * @return string
+ */
+function training_ajax_callback() {
+  ctools_include('ajax');
+  $output = '';
+  $output .= ctools_ajax_text_button('Click me!', 'training/ctools/ajax/callback-text', 'Click');
+
+  return $output;
+}
+
+/**
+ * Renders page callback for training/ctools/ajax/callback-text.
+ */
+function training_ajax_callback_text() {
+  drupal_add_js(drupal_get_path('module', 'training') . '/js/training.js');
+  $commands = array();
+  $commands[] = array(
+    'command' => 'reload',
+    'some_param' => rand(2, 4),
+  );
+  print ajax_render($commands);
+  exit;
+}
+
+/**
+ * Renders page callback for training/ctools/cleanstring.
+ *
+ * @return string
+ */
+function training_cleanstring_callback() {
+  ctools_include('cleanstring');
+  $output = array();
+  $output[] = ctools_cleanstring('some text /ё @12-5 [test]');
+  $output[] = ctools_cleanstring('some text', array(
+    'separator' => ' ',
+  ));
+  $output[] = ctools_cleanstring('some text', array(
+    'ignore words' => 'text',
+    'replacements' => array(
+      'some' => 'SomE',
+    ),
+  ));
+  $output[] = ctools_cleanstring('некоторый текст', array(
+    'transliterate' => TRUE,
+    'max length' => 15,
+  ));
+
+  return implode('<br />', $output);
+}
+
+/**
+ * Builds dynamic form.
+ *
+ * @param array $form
```

Comment:
remove @params and @return - no needs to declare these tags for hooks, regular form builders, validate and submit callbacks